reefer_monitoring/v1/reefer_v1.0.0-Beta-1.yaml

```diff
@@ -0,0 +1,703 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0-Beta-1
+  title: DCSA OpenAPI specification for Operational Reefer Monitoring
+  description: |-
+    Exchange data uniformly on data collected from live reefers between operational supply chain partners such as carriers, terminals and vessel operators.
+
+    For explanation to specific values or objects please refer to the [Information Model 2023.Q1](https://dcsa.org/wp-content/uploads/2022/02/DCSA-Information-Model-2023.Q1.pdf).
+
+    ## Poll model
+    Polling data can be done using
+
+        GET /v1/events
+
+    Calling this endPoint will return a list of events that fulfill the query parameters specified. The list returned is sorted in ascending (`ASC`) order based on the creation date-time of the event (`eventCreatedDateTime`). **This endPoint is mandatory to implement.**
+
+    For a changelog please click [here](https://github.com/dcsaorg/DCSA-OpenAPI/blob/master/reefer-monitoring/v1). Please also [create a GitHub issue](https://github.com/dcsaorg/DCSA-OpenAPI/issues/new) if you have any questions/comments.
+  contact:
+    name: Digital Container Shipping Association (DCSA)
+    email: info@dcsa.org
+    url: 'https://dcsa.org'
+  license:
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+    name: Apache 2.0
+paths:
+  /events:
+    get:
+      tags:
+        - Events
+      summary: Retrieve Reefer Information
+      description: Fetch the current monitoring data available for reefer container fleet
+      operationId: fetchReefer
+      parameters:
+        - $ref: ../../domain/event/event_domain_v3.1.0.yaml#/components/parameters/eventDateTime
```

Comment:
Wondering if this will work on SwaggerHub? You are using relative links instead of direct links?